pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.wdl

```diff
@@ -62,6 +63,21 @@ workflow ExomeGermlineSingleSample {
 
     Boolean skip_reblocking = false
     Boolean provide_bam_output = false
+
+    String cloud_provider
+  }
+
+  # docker images
+  String gatk_docker_gcp = "us.gcr.io/broad-gatk/gatk:4.5.0.0"
+  String gatk_docker_azure = "dsppipelinedev.azurecr.io/gatk_reduced_layers:latest"
```

Comment:
this docker is pointing to latest - do we have pinned version to pull? Just checking.